.vsts-dnceng.yml

```diff
@@ -154,7 +154,7 @@ phases:
 
     - task: PublishBuildArtifacts@1
       inputs:
-        PathtoPublish: '$(Build.SourcesDirectory)/Binaries/$(_configuration)/Logs'
+        PathtoPublish: '$(Build.SourcesDirectory)/Binaries/Debug/Logs'
```

Comment:
Why does this fix anything?

Reply:
Actually, I guess the better question is: why aren't we testing release for this?

Reply:
The `$(_configuration)` variable isn't defined for this job. It's hard coded to Debug. Only the jobs which do Release + Debug define this.

Reply:
@333fred Linux is still new. Focusing on getting Debug up + stable before we expand out to release.